lib/LANraragi/Utils/Routing.pm

```diff
@@ -152,15 +152,13 @@ sub apply_routes {
     $logged_in_api->put('/api/categories/:id/:archive')->to('api-category#add_to_category');
     $logged_in_api->delete('/api/categories/:id/:archive')->to('api-category#remove_from_category');
 
-    # Reading Group API
-    $public_api->put('/api/reading-group/test')->to('api-readinggroup#test');
-    $public_api->get('/api/reading-group')->to('api-readinggroup#get_reading_group_list');
-    $public_api->get('/api/reading-group/:id')->to('api-readinggroup#get_reading_group');
-    $public_api->put('/api/reading-group')->to('api-readinggroup#create_reading_group');
-    $public_api->delete('/api/reading-group/:id')->to('api-readinggroup#delete_reading_group');
-    $public_api->put('/api/reading-group/:id/archive')->to('api-readinggroup#update_archive_list');
-    $public_api->put('/api/reading-group/:id/:archive')->to('api-readinggroup#add_to_readinggroup');
-    # $public_api->delete('/api/reading-group/:id/:archive')->to('api-readinggroup#remove_from_reading_group');
+    # Tankoubon API
+    $public_api->get('/api/tankoubon')->to('api-tankoubon#get_tankoubon_list');
+    $public_api->get('/api/tankoubon/:id')->to('api-tankoubon#get_tankoubon');
+    $logged_in_api->put('/api/tankoubon')->to('api-tankoubon#create_tankoubon');
+    $logged_in_api->delete('/api/tankoubon/:id')->to('api-tankoubon#delete_tankoubon');
+    $logged_in_api->put('/api/tankoubon/:id/archive')->to('api-tankoubon#update_archive_list');
+    $logged_in_api->put('/api/tankoubon/:id/:archive')->to('api-tankoubon#add_to_tankoubon');
```

Comment:
Should we add `delete /api/tankoubon/:id/:archive` as well now?

Reply:
I still insist is not a good Idea to have individual remove, but I still added it anyway. It actually gets the score of the archive to remove, and modifiy everyone after it to reduce its score by 1. In case that is wanted that the add to to tank can insert on an specific position, it would need to do something similar, but changing the -1 to +1. 

Use it with wisdom.

Reply:
Yeah, not sure I'll use it in the web UI, but I think it's worth having for clients with an extra warning label on top. Thanks!